plugins.html

```diff
@@ -9,6 +9,7 @@
 	<h2>Install Plugins</h2>
 	<p>There are many plugins available to use, and installing them is easy. We have a plugin repository where you can find most plugins available for Cuberite. The User's Manual contains instructions on how to install a plugin.</p>
 	<a href="https://book.cuberite.org/#2.5" class="button">Installation Guide</a>
+	<a href="https://plugins.cuberite.org/" class="button">Plugin Repository</a>
 	<a href="https://forum.cuberite.org/forum-2.html" class="button">Download Plugins</a>
```

Comment:
The text here doesn't really make sense in connection with the previous line, could you change it?